Apromore-Core-Components/Apromore-Portal/src/main/resources/zk-label.properties

```diff
@@ -294,11 +295,11 @@ portal_unableSave_type = ERROR
 portal_langChanged_message = You need to reload the portal to see the effect
 portal_langChanged_title = Change language
 portal_langChanged_type = WARNING
-portal_itemsSelectedToMove_message = {0} item(s) have been selected to be moved
+portal_itemsSelectedToMove_message = {0} {0, choice, 1#item|1<items } have been selected to be moved
```

Comment:
That's slick; I've haven't seen this done before.  👌

Reply:
I have found it easy to handle conditional case.